### ISSUE TITLE
Tutorial Notebook about von Mises Fisher Distribution

### ISSUE THREAD

**Issue by AlexanderJuestel** [cgre-aachen/gempy#449]
**Is your feature request related to a problem? Please describe.**
Currently there is only tutorial 5.1 on how to add uncertainties for interface positions. A tutorial/notebook for adding uncertainties for orientations is missing. 

**Describe the solution you'd like**
I would like to create a notebook using the code by @elimh (fishdist.py) to better illustrate how uncertainties can be assigned to orientation values so that future users have better access to it

**Comment by Japhiolite** [comment]
Good idea, I have one (very very messy) notebook using MF Distributions here: https://github.com/Japhiolite/PCT-modelling/blob/master/notebooks/GemPy-PCT_Stochastic.ipynb  
The notebook is by no means useful for a Tutorial notebook. But maybe it's helpful as to show how to use certain features. 
That notebook heavily relies on code written by @elimh . I can also support you in writing that tutorial notebook, if you want.

**Comment by AlexanderJuestel** [comment]
Yeah, you are more than welcome to support :) 

I am currently going through the code to understand and reproduce how samples are drawn since I need that for my work as well. 

It is not urgent (for me) to have a notebook but it would be a nice addition I think. I can maybe set one up and push it to my development branch at some point and let you know :)

**Comment by AlexanderJuestel** [comment]
Thanks :)

Then I am sure, we will be able to put something together.
